src/pay/batch_payer.py

```diff
@@ -612,6 +612,9 @@ def attempt_single_batch(self, payment_items, op_counter, dry_run=None):
         for payment_item in payment_items:
 
             pymnt_amnt = payment_item.amount  # expected in micro tez
+            payment_item.transaction_fee = (
```

Comment:
There is no item "transaction_fee" in payment_item look at the reward_log.py

Reply:
I will fix it

Reply:
I think there needs to be a default one ^^ and if some additional fees occur they get added, or? (also one of the reason a test failed because it couldn't find the attribute `transaction_fee`)

Reply:
I pushed a fix. There was also a remaining transaction_fee in the code which should have not been there. Thanks for finiding this.